test-scripts/check.sh

```diff
@@ -48,7 +48,8 @@ all_pwd_sha1sums() {
 }
 
 elfshaker_sha1sums() {
-  "$elfshaker" list-files "$@" 2> /dev/null | awk '{print $1" "$3}' | sort -k2,2
+  # Note: Canonicalizes windows paths to unix paths for now.
+  "$elfshaker" list-files "$@" 2> /dev/null | awk '{print $1" "$3}' | sort -k2,2 | sed 's|\\|/|g'
```

Comment:
(in the future) we could make changes to print paths in Unix-style, regardless of what the native separator is. Git certainly doesn't use the Windows path separator when running on Windows.

Reply:
Sounds good to me (for the future :))